.nvmrc

```diff
@@ -0,0 +1 @@
+18
```

Comment:
would you mind if we made this an explicit version? at the moment, asdf-nodejs doesn't support installing "latest X version" (because node-build doesn't support it): https://github.com/asdf-vm/asdf-nodejs/issues/295

Reply:
Also, I was able to get this working with 16 to test with landscape-apps.

Reply:
Just saw the commit explicitly switching to 18. That's fine, we have workarounds we can use if we wanted to use this in landscape-apps today without changing the node version there.